editor/editor/style.scss

```diff
@@ -97,3 +112,8 @@
 		}
 	}
 }
+
+/* Hide footer */
+#wpfooter {
```

Comment:
Should we move this to the global stylesheet `main.scss`.

Should we add it under the `.toplevel_page_gutenberg` like other WP style tweaks (to avoid altering these style in other pages, if ever the editor style is mixed with all the WPAdmin stylesheets) https://github.com/WordPress/gutenberg/blob/add/polish/editor/assets/stylesheets/main.scss#L5